preload.js

```diff
@@ -23,8 +25,8 @@ window.renderTray = (unread) => {
 		canvas.height = logo.naturalHeight;
 
 		ctx.drawImage(logo, 0, 0);
-		if(unread !== undefined) {
-			unread = (Number(unread) > 99 ? 99 : unread);
+		if(unread > 0) {
+			unread = (unread > 99 ? 99 : unread);
```

Comment:
Can you display "99+" instead of "99" when Notification count crosses 99?

Reply:
I thought about that, but it would display 3 characters and won't fit in the badge.

Reply:
Oh okay, I will leave to 99 for now then an d merge this.